src/sessionTypes/auth0WebAuth.spec.js

```diff
@@ -196,7 +199,7 @@ describe('sessionTypes/Auth0WebAuth', function() {
 
       it("returns a fulfilled promise with the users's profile", function() {
         return expect(promise).to.be.fulfilled
-          .and.to.eventually.equal(expectedProfile);
+          .and.to.eventually.deep.equal(expectedProfile);
```

Comment:
I was not aware of `to.be.fulfilled.and.to.eventually.equal` 💯

Reply:
It is part of `chai-as-promised`